udev/systemd-udev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "udev",
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://github.com/systemd/systemd/archive/v242.tar.gz",
```

Comment:
I'm confident there is a ton of `cleanup` missing here.

Reply:
Yeah very likely :P I'm looking into it what's cleanable today or the next days, schedule gets a little bit stuffed right now :+1: